iroh-bytes/Cargo.toml

```diff
@@ -58,10 +59,11 @@ tokio = { version = "1", features = ["macros", "test-util"] }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
 rcgen = "0.12.0"
 rustls = { version = "0.21.0", default-features = false, features = ["quic"] }
+tempfile = "3.10.0"
 
 [features]
-default = ["flat-db"]
-flat-db = ["reflink-copy"]
+default = ["redb-db"]
+redb-db = ["reflink-copy", "redb"]
```

Comment:
maybe use a more generic term, like `disk-db`? not sure

Reply:
yeah, I was also thinking that the thing should a proper name. Like some character from a certain well known anime show or something...

Reply:
Maybe just filestore? The notable thing compared to the mem store is that it is using individual files. That is probably more permanent than using redb.

Reply:
yes, filestore makes sense to me

Reply:
want to do that rename before merging?

Reply:
Yeah.